patroni/dcs/__init__.py

```diff
@@ -500,7 +500,7 @@ def get_replication_slots(self, my_name, role):
                     value['type'] = 'logical' if value.get('database') and value.get('plugin') else 'physical'
 
                 if value['type'] == 'physical':
-                    if name != my_name:  # Don't try to create permanent physical replication slot for yourself
+                    if name != slot_name_from_member_name(my_name):  # Don't try to create permanent physical replication slot for yourself
```

Comment:
https://travis-ci.org/github/zalando/patroni/jobs/717510167#L1347:
```
patroni/dcs/__init__.py:503:121: E501 line too long (139 > 120 characters)
```

Reply:
Thanks. Fixing it now.